Sources/Controllers/TargetMenu/Editors/OAEditDescriptionViewController.m

```diff
@@ -217,7 +265,140 @@ - (IBAction)editClicked:(id)sender
+        if (cell == nil)
+        {
+            NSArray *nib = [[NSBundle mainBundle] loadNibNamed:[OAWebViewCell getCellIdentifier] owner:self options:nil];
+            cell = (OAWebViewCell *)[nib objectAtIndex:0];
+        }
+        if (cell)
+        {
+            NSNumber *inset = item[@"separatorInset"];
+            cell.separatorInset = UIEdgeInsetsMake(0, inset.floatValue, 0, 0);
+            cell.iconView.hidden = YES;
+            cell.arrowIconView.hidden = YES;
+            cell.webViewLeftConstraint.constant = 0;
+            cell.webViewRightConstraint.constant = 0;
+            cell.backgroundColor = UIColor.whiteColor;
+            cell.webView.navigationDelegate = self;
+            [cell.webView loadHTMLString:item[@"title"]  baseURL:nil];
+        }
+        return cell;
+    }
+    else if ([item[@"type"] isEqualToString:[OAIconTitleValueCell getCellIdentifier]])
+    {
+        OAIconTitleValueCell *cell = [tableView dequeueReusableCellWithIdentifier:[OAIconTitleValueCell getCellIdentifier]];
+        if (cell == nil)
+        {
+            NSArray *nib = [[NSBundle mainBundle] loadNibNamed:[OAIconTitleValueCell getCellIdentifier] owner:self options:nil];
+            cell = (OAIconTitleValueCell *) nib[0];
+            [cell showLeftIcon:NO];
+            [cell showRightIcon:NO];
+        }
+        if (cell)
+        {
+            NSNumber *inset = item[@"separatorInset"];
+            cell.separatorInset = UIEdgeInsetsMake(0, inset.floatValue, 0, 0);
+            cell.selectionStyle = UITableViewCellSelectionStyleDefault;
+            cell.textView.font = [UIFont systemFontOfSize:17. weight:UIFontWeightMedium];
+            cell.textView.textColor = UIColorFromRGB(color_primary_purple);
+            cell.textView.text = item[@"title"];
+            cell.descriptionView.hidden = YES;
+        }
+        return cell;
+    }
+
+    return nil;
+}
+
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    NSDictionary *item = _cellsData[indexPath.row];
+    if ([item[@"type"] isEqualToString:[OAWebViewCell getCellIdentifier]])
+        return _webViewHeight;
+    return UITableViewAutomaticDimension;
+}
+
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    NSDictionary *item = _cellsData[indexPath.row];
+    if ([item[@"type"] isEqualToString:[OAIconTitleValueCell getCellIdentifier]])
+        [self editClicked:nil];
+    [tableView deselectRowAtIndexPath:indexPath animated:YES];
+}
+
+#pragma mark - UITextViewDelegate
+
+- (void)textViewDidChange:(UITextView *)textView
+{
+    _textViewContent = textView.text;
+    [_tableView beginUpdates];
+    [_tableView endUpdates];
+}
+
+#pragma mark - WKNavigationDelegate
+
+- (void)webView:(WKWebView *)webView didFinishNavigation:(WKNavigation *)navigation
+{
+    BOOL shouldSkipFirstLoadingWithIncorrectData = _webViewReloadingsCount == 0;
```

Comment:
Why data is incorrect?

Reply:
First result: zero or small value. After reload - ok.
[img](https://www.dropbox.com/s/3ohhohq32claz7h/Screenshot%202022-01-27%20at%2014.32.28.png?dl=0)
[img](https://www.dropbox.com/s/hywrk7xtl9rzhja/Screenshot%202022-01-27%20at%2014.32.41.png?dl=0)

After rotarion sometimes returns old width and incorrect height.
[img](https://www.dropbox.com/s/0uu1pc5b2vkbhf0/Screenshot%202022-01-27%20at%2014.35.05.png?dl=0)